packages/serverless/src/apollo/app/collection-index.ts

```diff
@@ -96,7 +104,7 @@ const indexCreate = authorizeWrapper<TIndexCreateRequest, TIndexCreateResponse>(
   Joi.object({
     databaseName,
     collectionName,
-    index: CollectionIndex,
```

Comment:
Are you going to delete `CollectionIndex` joi as well?.

Reply:
Removed in common